cmd/helm/search/search_test.go

```diff
@@ -203,13 +203,21 @@ func TestSearchByName(t *testing.T) {
 			},
 		},
 		{
+			name:  "description upper search, two results",
+			query: "TWO",
+			expect: []*Result{
+				{Name: "testing/pinta"},
+				{Name: "ztesting/pinta"},
+			},
+		},
+		{
 			name:   "nothing found",
 			query:  "mayflower",
 			expect: []*Result{},
 		},
 		{
 			name:  "regexp, one result",
-			query: "th[ref]*",
+			query: "Th[ref]*",
```

Comment:
Looks much better, but I have to be picky here: this change in test case is unaffected by your logic changes. It's really minor, so I will approve this PR.